src/python/pants/backend/python/goals/run_pex_binary_integration_test.py

```diff
@@ -102,13 +104,16 @@ def test_entry_point(
 
 @pytest.mark.parametrize("execution_mode", [None, PexExecutionMode.VENV])
 @pytest.mark.parametrize("include_tools", [True, False])
+@pytest.mark.parametrize("venv_site_packages_copies", [True, False])
```

Comment:
I think it's better to leave this off. This test is semi-slow to to execute, and we're now doubling the tests we had before.

Reply:
Got it. Removed for now.